test/kubemark/cloudprovider.go

```diff
@@ -0,0 +1,342 @@
+/*
```

Comment:
I would suggest putting this file at `pkg/kubemark/cloudprovider.go` instead, as we usually don't check-in go code under this path but just have setup scripts and configs. A better place would have been `pkg/cloudproviders/providers/`, but it's being deprecated.